free-programming-books-pl.md

```diff
@@ -38,6 +38,7 @@
 
 ### Android
 
+* [O Androidzie ludzkim głosem](http://andrzejklusiewicz-android.blogspot.com/p/bezpatny-kurs-programowania-android-java.html) - Andrzej Klusiewicz
```

Comment:
It looks like this source is accessible via HTTPS, could we please use the https:// link instead?

```suggestion
* [O Androidzie ludzkim głosem](https://andrzejklusiewicz-android.blogspot.com/p/bezpatny-kurs-programowania-android-java.html) - Andrzej Klusiewicz
```